Content.Client/Kudzu/KudzuVisualizerComponent.cs

```diff
@@ -0,0 +1,10 @@
+namespace Content.Client.Kudzu
+{
+    [RegisterComponent]
+    public sealed class KudzuVisualizerComponent : Component
```

Comment:
```suggestion
    public sealed class KudzuVisualsComponent : Component
```

also rename the usages in system + yaml etc. this is just generally convention